class TestTry{
	public static void main(String args[])
	{
		int d=20/0;
		System.out.println("d"+d);
		System.out.println("Rest of the code");
	}
} 
// above program doesn't gives the output due to exception in thread main
